src/serverMain/kotlin/com/studo/campusqr/endpoints/UserManagement.kt

```diff
@@ -83,7 +84,8 @@ suspend fun AuthenticatedApplicationCall.editUser() {
 
   // Only ADMIN users can change the password of other users
   // Only ADMIN users can change user types
-  if (!user.isAdmin && (changedUserId != user._id || newUserType != null)) {
+  // Disallow changing editing user in demo mode
+  if ((!user.isAdmin && (changedUserId != user._id || newUserType != null)) || demoMode) {
```

Comment:
Why is editing users in demo mode disabled, but not creating/deleting users? And if we decide on that, we should also handle that on the UI-side.
But I think that we can allow everything in demo-mode if we provide on the login-view (and on the admin-settings-view) a "reset demo-data" button that can be executed even when not authenticated.